### PR TITLE
[SG-33416] Improve discoverability of the storybook preview link

### PR DESCRIPTION
**Pull request by gitstart-sourcegraph**
## Description

- Shell script to get Storybook preview url from Chromatic publish output and add it into App Preview section in PR description

## Test plan

Make sure all checks are passed in CI

## Refs

- [SourceGraph Issue](https://github.com/sourcegraph/sourcegraph/issues/33416)
- [GitStart ticket](https://app.gitstart.com/clients/sourcegraph/tickets/SG-33416)

## App preview:

- [Web](https://sg-web-contractors-sg-33416.onrender.com)
- [Storybook](https://5f0f381c0e50750022dc6bf7-swgfdyvcgk.chromatic.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.

### CODE REVIEW

File: client/wildcard/src/components/Button/Button.tsx [hunk 1]
```diff
@@ -34,7 +34,9 @@ export interface ButtonProps extends React.ButtonHTMLAttributes<HTMLButtonElemen
 }
 
 /**
- * Simple button.
+ * Simple button
+ *
+ * Faking client changes
```
Comment (gitstart-sourcegraph) [inline client/wildcard/src/components/Button/Button.tsx:39]: Revert this at the final commit

File: enterprise/dev/ci/internal/ci/pipeline.go [hunk 2]
```diff
@@ -95,9 +95,10 @@ func GeneratePipeline(c Config) (*bk.Pipeline, error) {
 
 			// Do not create client PR preview if Go or GraphQL is changed to avoid confusing
 			// preview behavior, because only Client code is used to deploy application preview.
-			if !c.Diff.Has(changed.Go) && !c.Diff.Has(changed.GraphQL) {
-				ops.Append(prPreview())
-			}
+			// Disable these checks temporarily
+			// if !c.Diff.Has(changed.Go) && !c.Diff.Has(changed.GraphQL) {
+			ops.Append(prPreview())
+			// }
```
Comment (gitstart-sourcegraph) [inline enterprise/dev/ci/internal/ci/pipeline.go:101]: Revert this at the final commit

**Review by valerybugakov — APPROVED** [review]
LGTM, thanks!

**Comment by gitstart-sourcegraph** [comment]
@valerybugakov should we merge this PR or wait from @sourcegraph/dev-experience

**Comment by valerybugakov** [comment]
@gitstart-sourcegraph, let's merge this PR since it doesn't affect the CI pipeline and is similar to the logic we reviewed for the app preview.

**Pull request merged**